### DISCUSSION TITLE
false positives - shadows and clouds

### DISCUSSION THREAD

**Discussion by jaredmcqueen** [blakeblackshear/frigate#6791] [Ask A Question]
Before you ask, I have looked at this thread:
https://github.com/blakeblackshear/frigate/discussions/6574

I have also read and understand the docs for false positives:
https://docs.frigate.video/guides/false_positives

That being said - every night I get setting sun shadows on my driveway and I cannot for the life of me figure out how to filter them out.  Also sometimes I get clouds.  All things considered, a handful of false positives a day isn't _that_ bad, but still -- would like to see how ya'll deal with them.

![image](https://github.com/blakeblackshear/frigate/assets/3491953/9691ef1b-183f-4c4c-b39c-284f35d2283c)
![image](https://github.com/blakeblackshear/frigate/assets/3491953/d6a1ffda-2911-4582-9830-e03ba347cfaa)
![image](https://github.com/blakeblackshear/frigate/assets/3491953/8168c21d-cf18-4ab9-baaf-3bebb200f243)

My current relevant tuning is:
```yaml
objects:
  filters:
    person:
      min_area: 20000
      min_score: 0.6
      threshold: 0.7

```

running 0.12.1.

Would using other training models be better for my driveway cam?  Does anyone have success with other models?  Thanks!

**Comment by NickM-27** [comment]
A model like Frigate+ that is trained on relevant images (images actually taken with security cameras) will be better than the default COCO model. There aren't many options out there and frigate+ is still under development. 

The only thing I can think to filter these out would be a max ratio filter but that would block any people who are horizontal / laying down (may not be a concern but figured I'd mention it)

**Reply by jaredmcqueen** [reply]
thanks for the heads up.  Will stay tuned to frigate+ to see when models are available.

**Comment by LaurenceGough** [comment]
Bit of a bump but in this example I'd setup an object filter zone for that area if your car is always parked in roughly the same area.  At least motion masks on reflective surfaces such as the car windows - it'll help reduce CPU usage from the clouds as well.
